src/medCore/gui/commonWidgets/medSlider.cpp

```diff
@@ -15,25 +15,35 @@
 
 void medSlider::addTick(int position)
 {
-    if(!ticksList.contains(position))
+    if( (tickCount() == 0) || !ticksList.contains(position))
     {
         ticksList.append(position);
     }
 }
 
 int medSlider::tickCount()
 {
-   return ticksList.count();
+    if (!ticksList.toVector().isEmpty())
```

Comment:
yes, it can be interesting 'cause Qt doc says:
> To make QList as efficient as possible, its member functions don't validate their input before using it. Except for isEmpty(), member functions always assume the list is not empty. Member functions that take index values as parameters always assume their index value parameters are in the valid range. [...]
To avoid failures when your list can be empty, call isEmpty() before calling other member functions.